samples/bluetooth/broadcaster_multiple/boards/nrf52_bsim.conf

```diff
@@ -0,0 +1,18 @@
+# This is a sample Zephyr Controller configuration to test PDU chaining.
+
+# Zephyr Controller will use chain PDUs when AD data length > 191 bytes
+# - 31 bytes will use 22 bytes for the default name in this sample plus 9 bytes
+#   for manufacturer data
+# - 191 bytes will use 22 bytes for the default name in this sample plus 169
+#   bytes for manufacturer data
+# - 277 bytes will use 22 bytes for the default name in this sample plus 255
+#   bytes for manufacturer data
+# CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=192
+
+# Enabling PDU chaining
+# CONFIG_BT_CTLR_ADVANCED_FEATURES=y
+# CONFIG_BT_CTLR_ADV_AUX_PDU_BACK2BACK=y
```

Comment:
Shouldn't the default configuration make it possible to perform chaining?

Reply:
Can you deduct if chaining is required from `CONFIG_BT_CTLR_ADV_DATA_LEN_MAX`?

Reply:
I guess so, let me try if setting the values in Kconfig and update the PR, if I succeed.